standard/types.md

```diff
@@ -537,11 +542,11 @@ type_arguments
     ;   
 
 type_argument
-    : type
+    : type nullable_type_attribute?
```

Comment:
Correct. this is so `string?` can be used. 

I think we already do what you say:  The grammar for NRTs and Nullable Value types specifies them as a `type` followed by the `nullable_type_attribute`.

Reply:
But `type` already matches `string?` because `type` includes `reference_type` as an option, and that includes `nullable_reference_type` as an option.

So I think `Foo<string?>` is already covered without this change, but `Foo<T?>` wouldn't be.

(With this change, `string??` is a valid type argument, because the `type` production would permit `string?` and then there's the optional `nullable_type_argument`.)

Reply:
Hmm. Are you thinking that something should be added in 8.5 (type parameters) so that a *type_parameter* could be either an *identifier* or *identifier* *nullable_type_attribute*?  That would allow `T` and `T?`.  However, I think that would also allow `string??` when the type argument was `string?` and the type parameter was `T`.

We made need help from Nigel, or a statement in prose about that happens here.

Reply:
The grammar has been updated, so this is no longer an issue.